kube-runtime/src/controller/mod.rs

```diff
@@ -37,7 +37,7 @@ mod future_hash_map;
 mod runner;
 
 #[derive(Debug, Error)]
-pub enum Error<ReconcilerErr: std::error::Error + 'static, QueueErr: std::error::Error + 'static> {
+pub enum Error<ReconcilerErr: 'static, QueueErr: 'static> {
```

Comment:
oh, huh, does this actually work :thinking:

Reply:
Thiserror automatically adds `...: Error` bounds when generating `impl Error`, so these bounds were not necessary.